feature-libs/asm/core/facade/asm-query.service.ts

```diff
@@ -0,0 +1,53 @@
+import { Injectable } from '@angular/core';
+import {
+  AsmFacadeService,
+  CustomerListsPage,
+  CustomerSearchOptions,
+  CustomerSearchPage,
+} from '@spartacus/asm/root';
+import {
+  Command,
+  CommandService,
+  CommandStrategy,
+  Query,
+  QueryService,
+  QueryState,
+} from '@spartacus/core';
+import { Observable } from 'rxjs';
+import { AsmConnector } from '../connectors/asm.connector';
+
+@Injectable()
+export class AsmQueryService implements AsmFacadeService {
```

Comment:
`AsmQueryService` is not a good name. We can just create queries in facade service.